docs/source/custom-collections.rst

```diff
@@ -420,9 +420,9 @@ In pseudocode this looks like:
 Adding the Core Dask Methods to Your Class
 ------------------------------------------
 
-Defining the above interface will allow your object to used by the core dask
-functions (``dask.compute``, ``dask.persist``, ``dask.visualize``, etc...). To
-add corresponding method versions of these subclass from
+Defining the above interface will allow your object to used by the core Dask
+functions (``dask.compute``, ``dask.persist``, ``dask.visualize``, etc.). To
+add corresponding method versions of these subclasses from
```

Comment:
> To add corresponding method versions of these subclasses from ``dask.base.DaskMethodsMixin,`` which adds implementations of compute, persist, and visualize based on the interface above

This sentence is confusing and I don't know to fix it properly. Any suggestions?

Reply:
Here `subclass` is a verb, and was probably fine as is.